src/components/ContentSidebar/__tests__/ContentSidebar-test.js

```diff
@@ -295,4 +295,132 @@ describe('components/ContentSidebar/ContentSidebar', () => {
             instance.onCommentCreate('text', false, jest.fn(), testErrorCallback);
         });
     });
+
+    describe('onTaskCreateSuccess()', () => {
+        let instance;
+        let wrapper;
+        beforeEach(() => {
+            wrapper = getWrapper();
+            instance = wrapper.instance();
+        });
+
+        test('should not add to the task entries state if there is none', () => {
+            instance.setState({ tasks: undefined });
+            instance.setState = jest.fn();
+
+            instance.onTaskCreateSuccess({});
+
+            expect(instance.setState).not.toBeCalled();
+        });
+
+        test('should add the task to the task entries state', () => {
```

Comment:
I think this and the below test can be combined into 1 test. Just add the assertion and change the test name

Reply:
done